common/lib/container-definitions/src/loader.ts

```diff
@@ -87,7 +87,7 @@ export interface IContainerEvents extends IEvent {
     (event: "codeDetailsProposed", listener: (codeDetails: IFluidCodeDetails, proposal: IPendingProposal) => void);
     (event: "contextChanged", listener: (codeDetails: IFluidCodeDetails) => void);
     (event: "disconnected" | "attached", listener: () => void);
-    (event: "closed", listener: (error?: ICriticalContainerError) => void);
+    (event: "closed", listener: (reason: string, error?: ICriticalContainerError) => void);
```

Comment:
Are you going to backport this to 0.39 first?  You must have done symlinking here to build up this PR?

Reply:
Also... this signature looks backwards of how it's used in container.ts

Reply:
No, I did not do symlinking, I missed that it's in definition packages - compiler does not see it (i.e. can't match to emit usage).
I'll make it last arg and optional, for simplicity